pkg/Microsoft.Windows.Compatibility/tests/Microsoft.Windows.Compatibility.Validation.csproj

```diff
@@ -1,19 +1,59 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="../../../eng/Versions.props" />
+
   <ItemGroup>
     <PackageReference Include="Microsoft.Windows.Compatibility" Version="$(CompatibilityPackageVersion)" />
-   </ItemGroup>
-   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
-    <PackageReference Include="Microsoft.Private.CoreFx.NETCoreApp" Version="$(PrivateCorefxPackageVersion)" />
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="3.0.0-*" />
-    <PackageReference Include="Microsoft.DesktopUI.App" Version="3.0.0-*" />
-    <PackageReference Include="System.Windows.Extensions" Version="4.6.0-*" />
-   </ItemGroup>
-   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
```

Comment:
I don't think we should preserve this working for older frameworks as as soon as we ship we just generate an static drop.  I've never had to use this for an older TFM. Maybe let's just leave the netcoreapp3.0 piece and then everytime we re-brand we need to update if necessary.